RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -107,6 +110,16 @@ struct ManageSubscriptionsView: View {
         .task {
             await loadInformationIfNeeded()
         }
+        .toolbar {
```

Comment:
using `.topBarLeading` is pretty subjective. not sure if the placement, visibility, etc. needs to be customizable outside of the dismiss text

Reply:
I think this makes sense.